ethereum/test/test_incentivized_outbound_channel.js

```diff
@@ -13,18 +14,25 @@ require("chai")
 
 contract("IncentivizedOutboundChannel", function (accounts) {
   // Accounts
+  const owner = accounts[0];
   const userOne = accounts[1];
+  const userTwo = accounts[2];
   const testPayload = ethers.utils.formatBytes32String("arbitrary-payload");
+  const relayFee = process.env.RELAY_FEE;
+  const feeController = process.env.FEE_CONTROLLER
 
   describe("deployment and initialization", function () {
     beforeEach(async function () {
-      this.channel = await IncentivizedOutboundChannel.new();
+      this.channel = await IncentivizedOutboundChannel.new(relayFee, feeController);
     });
   });
 
   describe("send", function () {
     beforeEach(async function () {
-      this.channel = await IncentivizedOutboundChannel.new();
+      this.channel = await IncentivizedOutboundChannel.new(relayFee, feeController);
+      const mock = await MockContract.new()
```

Comment:
You shouldn't need to use mock for DOTApp. You can make use of the `deployAppContractWithChannels` helper by copying over these lines from` test_dot_app.js`:
```
      this.erc1820 = await singletons.ERC1820Registry(owner);
      [this.channels, this.app] = await deployAppContractWithChannels(DOTApp);
      this.token = await Token.at(await this.app.token());
```

But for that to work, you either need remove the constructor parameters from IncentivizedOutboundChannel, or update the `deployAppContractWithChannels` helper to pass the constructor parameters when creating the channel.
https://github.com/Snowfork/polkadot-ethereum/pull/294#discussion_r584705075

Its probably easiest to remove the constructor parameters.

Reply:
Ah I see you updated deployAppContractWithChannels already. sorry, I missed that. Did deployAppContractWithChannels  still not work for you after that?